### ISSUE TITLE
IllegalArgumentException on start-up

### ISSUE THREAD

**Issue by SlimeDog** [RockinChaos/ItemJoin#234]
Spigot 1.14.4 latest (Jenkins 2474)
ItemJoin 5.0.5-SNAPSHOT-b310

Menu fails to load. Both config.yml and items.yml are unchanged from build 309.
- http://blackdog.straight8.com/drops/config.yml
- http://blackdog.straight8.com/drops/items.yml
```
[03:15:42] [Server thread/INFO]: [ItemJoin] Enabling ItemJoin v5.0.5-SNAPSHOT-b310
[03:15:42] [Server thread/ERROR]: Error occurred while enabling ItemJoin v5.0.5-SNAPSHOT-b310 (Is it up to date?)
java.lang.IllegalArgumentException: Material cannot be null
        at org.apache.commons.lang.Validate.notNull(Validate.java:192) ~[spigot-1.14.4.jar-2019-09-18-0649:git-Spigot-94af569-a3fdef8]
        at org.bukkit.inventory.ItemStack.<init>(ItemStack.java:70) ~[spigot-1.14.4.jar-2019-09-18-0649:git-Spigot-94af569-a3fdef8]
        at org.bukkit.inventory.ItemStack.<init>(ItemStack.java:58) ~[spigot-1.14.4.jar-2019-09-18-0649:git-Spigot-94af569-a3fdef8]
        at me.RockinChaos.itemjoin.utils.Legacy.newLegacyItemStack(Legacy.java:72) ~[?:?]
        at me.RockinChaos.itemjoin.handlers.ItemHandler.getItem(ItemHandler.java:116) ~[?:?]
        at me.RockinChaos.itemjoin.utils.UI.<init>(UI.java:46) ~[?:?]
        at me.RockinChaos.itemjoin.handlers.ConfigHandler.generateData(ConfigHandler.java:46) ~[?:?]
        at me.RockinChaos.itemjoin.ItemJoin.onEnable(ItemJoin.java:14) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[spigot-1.14.4.jar-2019-09-18-0649:git-Spigot-94af569-a3fdef8]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:352) [spigot-1.14.4.jar-2019-09-18-0649:git-Spigot-94af569-a3fdef8]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:417) [spigot-1.14.4.jar-2019-09-18-0649:git-Spigot-94af569-a3fdef8]
        at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugin(CraftServer.java:461) [spigot-1.14.4.jar-2019-09-18-0649:git-Spigot-94af569-a3fdef8]
        at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugins(CraftServer.java:375) [spigot-1.14.4.jar-2019-09-18-0649:git-Spigot-94af569-a3fdef8]
        at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:449) [spigot-1.14.4.jar-2019-09-18-0649:git-Spigot-94af569-a3fdef8]
        at net.minecraft.server.v1_14_R1.DedicatedServer.init(DedicatedServer.java:266) [spigot-1.14.4.jar-2019-09-18-0649:git-Spigot-94af569-a3fdef8]
        at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:775) [spigot-1.14.4.jar-2019-09-18-0649:git-Spigot-94af569-a3fdef8]
        at java.lang.Thread.run(Thread.java:834) [?:?]
```
It would be good to detect and report more information about the offending material and context, so it can be addressed.

**Comment by RockinChaos** [comment]
Whoops! Tweaked something that I didn't mean to haha. 
Resolved; http://ci.craftationgaming.com/job/ItemJoin/311/

**Comment by SlimeDog** [comment]
A brief does-it-enable test before uploading would be good. :)

Build 311 fixed the issue.

**Comment by RockinChaos** [comment]
Yeah, I really have been dropping the ball lol.
I have been pretty much programming in-class and I don't have a local testing server on my laptop as it really struggles to (even start). Sorry about that, ill try to make sure I push the code to my main desktop computer before I upload.

**Comment by RockinChaos** [comment]
This bug has been officially fixed in the release of ItemJoin v5.0.5.
You can grab it here; https://www.spigotmc.org/resources/itemjoin.12661/download?version=295652

Please read the changelog as the documentation has not been updated yet;
https://github.com/RockinChaos/ItemJoin/wiki/Recent-Changes
